websites/M/Minecraft Menu/dist/metadata.json

```diff
@@ -0,0 +1,45 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "Dark_Ville",
+		"id": "638080361179512853"
+	},
+	"service": "Minecraft Menu",
+	"description": {
+		"nl": "Minecraft Menu is een plek om de Beste Minecraft Server te vinden in 2022 voor Minecraft.",
+		"en": "Minecraft Menu is a place to find the Best Minecraft Servers in 2022 on Minecraft."
+	},
+	"url": [
+		"minecraft.menu",
+		"www.minecraft.menu"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/BlAb5Pg.png",
```

Comment:
This scales poorly on the website, resize the image to be as close to 512x512 without messing up the quality